cmds/live.c

```diff
@@ -104,15 +104,23 @@ int command_live(int argc, char *argv[], struct opts *opts)
 	};
 	int ret;
 
+	tmp_dirname = template;
 	umask(022);
 	fd = mkstemp(template);
-	if (fd < 0)
-		pr_err("cannot create temp name");
+	if (fd < 0) {
+		if (errno != EPERM)
+			pr_err("cannot access to /tmp");
+
+		fd = mkstemp(template + sizeof("/tmp/") - 1);
+
+		if (fd < 0)
+			pr_err("cannot create temp name");
+		tmp_dirname = template + sizeof("/tmp/") - 1;
+	}
 
 	close(fd);
 	unlink(template);
```

Comment:
You could either overwrite template or use tmp_dirname here (and below).

Reply:
Modified to tmp_dirname += sizeof("/tmp/") - 1;

Reply:
Thanks. I've fixed now